router/userRouter.js

```diff
@@ -13,4 +13,7 @@ router.post("/upload-image", [[header("uid").notEmpty()], validateFieldErrors, u
 router.post("/tag-user", [validateTagUserFields, validateFieldErrors], postController.tagUser)
 router.post("/remove-tag-user", [validateTagUserFields, validateFieldErrors], postController.removeTagUser)
 router.get("/all-posts/:page", [[param("page").isInt().custom((val) => { return val > 0 }).withMessage("page cant be less then one"), query("limit").isInt().custom((val) => { return val > 0 }).withMessage("limit cant be less then one")], validateFieldErrors], postController.allPosts)
+router.post("/add-comment", [[body("commentTitle").notEmpty(), body("userUid").notEmpty(), body("postUid").notEmpty()], validateFieldErrors], postController.addCommentOnPost)
```

Comment:
why you again -2 creating validator functions in route? they should be in middleware file @ashok-choudhary-0